.azure/docs-linux.yml

```diff
@@ -9,6 +9,7 @@ jobs:
 
     variables:
       PIP_CACHE_DIR: $(Pipeline.Workspace)/.pip
+      RUST_DEBUG: 1
```

Comment:
This doesn't actually matter. Tox is using editable installs. And even after my PR to turn that off for docs (to fix macOS), we'll still have Tox set RUST_DEBUG.

But, I figure it doesn't hurt to be consistent here. Avoids us accidentally losing this in the future if someone changes tox.ini without realizing the implications.